readme.md

```diff
@@ -25,7 +25,7 @@
 - Warns about the possibility of extraneous files being published
 - See exactly what will be executed with [preview mode](https://github.com/sindresorhus/np/issues/391), without pushing or publishing anything remotely
 - Supports [GitHub Packages](https://github.com/features/packages)
-- Supports npm 9+, Yarn (Classic and Berry), and pnpm 8+
+- Supports npm 9+, Yarn (Classic and Berry), npm 8+, and Bun
```

Comment:
```suggestion
- Supports npm 9+, Yarn (Classic and Berry), pnpm 8+, and Bun
```

Reply:
Fixed in https://github.com/sindresorhus/np/pull/760